### PR TITLE
update `zh-CN` to i10n patch 1

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

**Comment by bsmth** [comment]
Thanks for the PR! Is there a description available somewhere so we know the scope of the changes? (will there be a patch 2 or follow-up PRs?)

File: l10n/css.json [hunk 1]
```diff
@@ -150,15 +159,17 @@
...
   },
   "allElementsButNonReplacedAndTableColumns": {
     "de": "alle Elemente außer nicht ersetzte Inlineelemente, Tabellenspalten und Spaltengruppen",
     "en-US": "all elements but non-replaced inline elements, table columns, and column groups",
     "es": "elementos de bloque o remplazados",
     "fr": "tous les éléments sauf les éléments en ligne non remplacés, les colonnes de tableaux et les groupes de colonnes",
     "ja": "非置換インライン要素、テーブルの列、列グループを除くすべての要素",
-    "ru": "все элементы, кроме незаменяемых строчных элементов, табличных колонок и групп колонок"
+    "ru": "все элементы, кроме незаменяемых строчных элементов, табличных колонок и групп колонок",
+    "zh-CN": "all elements but non-replaced inline elements, table columns, and column groups"
```
Comment (bsmth) [inline l10n/css.json:172]: This is `en-US` duplicate

**Comment by T34-active** [comment]
> Thanks for the PR! Is there a description available somewhere so we know the scope of the changes? (will there be a patch 2 or follow-up PRs?)谢谢你的公关！是否有可用的描述，以便我们知道更改的范围？(是否会有补丁2或后续公共关系?)

yes, Subsequent possible submission of patch 2

File: l10n/css.json (hunk 1 above)
Comment (T34-active) [reply l10n/css.json]: done

**Review by bsmth — APPROVED** [review]
This looks good to me when I used machine translation, are you able to get a zh-CN reviewer to have a look, also? Thank you!

**Comment by T34-active** [comment]
OK /cc @yin1999 ,this is zh-CN reviewer

**Comment by yin1999** [comment]
Hey @T34-active. Thanks for the translation. Actually, I've finished some translation in my fork (see: yin1999/data#1). I've reviewed some translation you added in this PR, and felt that some of the translations could be improved by yourself first. For example, unnecessary quantifier translations (e.g. `一个`) and unnecessary characters in Chinese (e.g. `绝对长度` - `绝对 <length>`, the character `的` is not necessary).

Please do it by yourself first, and ping me again after you think it's ready for review by the `l10n-zh` team.

File: l10n/css.json [hunk 2]
```diff
@@ -4,56 +4,64 @@
     "en-US": "absolute {{cssxref(\"length\")}}",
     "fr": "une longueur (type {{cssxref(\"length\")}}) absolue",
     "ja": "絶対的な{{cssxref(\"length\", \"長さ\")}}",
-    "ru": "абсолютная {{cssxref(\"length\")}}"
+    "ru": "абсолютная {{cssxref(\"length\")}}",
+    "zh-CN": "绝对的 {{cssxref(\"length\")}}"
```
Comment (T34-active) [inline l10n/css.json:8]:
```suggestion
    "zh-CN": "绝对 {{cssxref(\"length\")}}"
```

File: l10n/css.json [hunk 3]
```diff
@@ -4,56 +4,64 @@
...
   },
   "absoluteLength0ForNone": {
     "de": "eine absolute Länge; falls das Schlüsselwort <code>none</code> angegeben wurde, ist der berechnete Wert <code>0</code>",
     "en-US": "an absolute length; if the keyword <code>none</code> is specified, the computed value is <code>0</code>",
     "fr": "une longueur absolue ; si le mot-clé <code>none</code> est spécifié, la valeur calculée sera <code>0</code>",
     "ja": "絶対的な長さ、キーワード <code>none</code> が指定されると計算値は <code>0</code> になる",
-    "ru": "абсолютная длина; если указано ключевое слово <code>none</code>, вычисленное значение - <code>0</code>"
+    "ru": "абсолютная длина; если указано ключевое слово <code>none</code>, вычисленное значение - <code>0</code>",
+    "zh-CN": "一个绝对长度；如果指定关键词为 <code>none</code>，则计算值为 <code>0</code>"
```
Comment (T34-active) [inline l10n/css.json:16]:
```suggestion
    "zh-CN": "绝对长度；如果指定关键词为 <code>none</code>，则计算值为 <code>0</code>"
```

File: l10n/css.json [hunk 4]
```diff
@@ -1575,7 +1616,8 @@
     "en-US": "a simple list of ",
     "fr": "une simple liste de ",
     "ja": "単純なリスト: ",
-    "ru": "простой список из "
+    "ru": "простой список из ",
+    "zh-CN": "一个简单的列表："
```
Comment (T34-active) [inline l10n/css.json:1620]:
```suggestion
    "zh-CN": "简单的列表："
```

**Review by T34-active — COMMENTED** [review]
fix content

File: l10n/css.json [hunk 5]
```diff
@@ -691,7 +719,8 @@
     "en-US": "elements with <code>{{cssxref(\"display\")}}: marker;</code>",
     "fr": "éléments avec <code>{{cssxref(\"display\")}}: marker;</code>",
     "ja": "<code>{{cssxref(\"display\")}}: marker;</code> の要素",
-    "ru": "элементы с <code>{{cssxref(\"display\")}}: marker;</code>"
+    "ru": "элементы с <code>{{cssxref(\"display\")}}: marker;</code>",
+    "zh-CN": "使用 <code>{{cssxref(\"display\")}}: marker;</code> 的元素"
```
Comment (yin1999) [inline l10n/css.json:723]:
```suggestion
    "zh-CN": "设置了 <code>{{cssxref(\"display\")}}: marker;</code> 的元素"
```

File: l10n/css.json [hunk 6]
```diff
@@ -759,7 +790,8 @@
     "en-US": "floats",
     "fr": "flottants",
     "ja": "浮動要素",
-    "ru": "плавает"
+    "ru": "плавает",
+    "zh-CN": "浮动"
```
Comment (yin1999) [inline l10n/css.json:794]: context: https://developer.mozilla.org/zh-CN/docs/Web/CSS/shape-image-threshold

```suggestion
    "zh-CN": "浮动元素"
```

File: l10n/css.json [hunk 7]
```diff
@@ -780,14 +812,16 @@
     "en-US": "for {{xref_csslength}} the absolute value, otherwise a percentage",
     "fr": "pour une valeur de type {{xref_csslength}} sa valeur absolue, sinon un pourcentage",
     "ja": "{{xref_csslength}} の場合は絶対的な値、それ以外の場合はパーセント値",
-    "ru": "для {{xref_csslength}} абсолютное значение, иначе процент"
+    "ru": "для {{xref_csslength}} абсолютное значение, иначе процент",
+    "zh-CN": "对于 {{xref_csslength}} 取其绝对值，反之为百分比"
```
Comment (yin1999) [inline l10n/css.json:816]:
```suggestion
    "zh-CN": "对于 {{xref_csslength}} 则为绝对值，否则为百分比值"
```

File: l10n/css.json [hunk 8]
```diff
@@ -1575,7 +1616,8 @@
     "en-US": "a simple list of ",
     "fr": "une simple liste de ",
     "ja": "単純なリスト: ",
-    "ru": "простой список из "
+    "ru": "простой список из ",
+    "zh-CN": "简单的列表："
```
Comment (yin1999) [inline l10n/css.json:1620]: context: https://developer.mozilla.org/zh-CN/docs/Web/CSS/transform-origin#%E8%A7%84%E8%8C%83

```suggestion
    "zh-CN": "由以下类型组成的简单列表："
```

File: l10n/css.json [hunk 9]
```diff
@@ -1618,38 +1660,44 @@
     "en-US": "table-caption elements",
     "fr": "éléments qui sont des légendes de tableaux",
     "ja": "表のキャプション要素",
-    "ru": "заголовки таблиц"
+    "ru": "заголовки таблиц",
+    "zh-CN": "表格说明文字元素"
```
Comment (yin1999) [inline l10n/css.json:1664]: https://developer.mozilla.org/zh-CN/docs/Web/HTML/Element/caption

```suggestion
    "zh-CN": "表格标题元素"
```

File: l10n/css.json [hunk 10]
```diff
@@ -1618,38 +1660,44 @@
...
     "fr": "des éléments <code>table</code> and <code>inline-table</code>",
     "ja": "<code>table</code> および <code>inline-table</code> 要素",
-    "ru": "<code>table</code> и <code>inline-table</code> элементы"
+    "ru": "<code>table</code> и <code>inline-table</code> элементы",
+    "zh-CN": "<code>table</code> 和 <code>inline-table</code> 元素"
   },
   "textAndBlockContainers": {
-    "en-US": "text and block containers"
+    "en-US": "text and block containers",
+    "zh-CN": "文本和区块内容"
```
Comment (yin1999) [inline l10n/css.json:1684]:
```suggestion
    "zh-CN": "文本和区块容器"
```

File: l10n/css.json [hunk 11]
```diff
@@ -1728,32 +1780,37 @@
...
+    "zh-CN": "视觉"
   },
   "visualInContinuousMediaNoEffectInOverflowColumns": {
     "de": "<code>visual</code>, aber in kontinuierlichen Medien hat sie keinen Effekt in überlaufenden Spalten",
     "en-US": "<code>visual</code>, but, in continuous media, has no effect in overflow columns",
     "fr": "<code>visual</code>, mais n'a pas d'effet sur le débordement des colonnes pour un média continu",
     "ja": "<code>visual</code>。ただし連続メディアではオーバーフローした列に効果なし",
-    "ru": "<code>visual</code>, но в сплошной среде, не имеет никакого эффекта при переполнении колонок"
+    "ru": "<code>visual</code>, но в сплошной среде, не имеет никакого эффекта при переполнении колонок",
+    "zh-CN": "<code>visual</code>, but, in continuous media, has no effect in overflow columns"
```
Comment (yin1999) [inline l10n/css.json:1792]:
```suggestion
    "zh-CN": "<code>visual</code>，但在连续媒体中，对溢出的列没有影响"
```

File: l10n/css.json [hunk 12]
```diff
@@ -1728,32 +1780,37 @@
...
+    "ru": "XUL {{XULElem(\"image\")}} элементы и {{cssxref(\":-moz-tree-image\")}}, {{cssxref(\":-moz-tree-twisty\")}} и {{cssxref(\":-moz-tree-checkbox\")}} псевдоэлементы. <strong>Заметьте:</strong> <code>-moz-image-region</code> работает только с элементами {{XULElem(\"image\")}}, где иконка определяется использованием {{cssxref(\"list-style-image\")}}. Это не будет работать с XUL <code>&lt;image src=\"url\" /&gt;</code>.",
+    "zh-CN": "XUL {{XULElem(\"image\")}} 元素和 {{cssxref(\":-moz-tree-image\")}}、{{cssxref(\":-moz-tree-twisty\")}} 和 {{cssxref(\":-moz-tree-checkbox\")}} 伪元素。<strong>注意：</strong> <code>-moz-image-region</code> 属性仅适用于通过 {{cssxref(\"list-style-image\")}} CSS 属性来指定图标的 XUL {{XULElem(\"image\")}} 元素。它不适用于使用 XUL <code>&lt;image src=\"url\" /&gt;</code> 格式的XUL图像标签。"
```
Comment (yin1999) [inline l10n/css.json:1800]:
```suggestion
    "zh-CN": "XUL {{XULElem(\"image\")}} 元素以及 {{cssxref(\":-moz-tree-image\")}}、{{cssxref(\":-moz-tree-twisty\")}} 和 {{cssxref(\":-moz-tree-checkbox\")}} 伪元素。<strong>注意：</strong><code>-moz-image-region</code> 属性仅适用于通过 {{cssxref(\"list-style-image\")}} 来指定图标的 XUL {{XULElem(\"image\")}} 元素。它不适用于 XUL 标签为 <code>&lt;image src=\"url\" /&gt;</code> 的元素。"
```

File: l10n/css.json [hunk 13]
```diff
@@ -1728,32 +1780,37 @@
...
     "pl": "tak",
-    "ru": "да"
+    "ru": "да",
+    "zh-CN": "是"
   },
   "zoomForTheTopLevelNoneForTheRest": {
     "en-US": "zoom for the top level element, none for all other elements",
-    "ja": "最上位要素では zoom、その他の要素では none"
+    "ja": "最上位要素では zoom、その他の要素では none",
+    "zh-CN": "顶层元素的缩放，所有其他元素无缩放"
```
Comment (yin1999) [inline l10n/css.json:1814]: https://learn.microsoft.com/en-us/previous-versions/hh771891(v=vs.85)

```suggestion
    "zh-CN": "顶层元素为 zoom，所有其他元素为 none"
```

**Review by yin1999 — COMMENTED** [review]
/cc @jasonren0403

File: l10n/css.json [hunk 14]
```diff
@@ -365,14 +381,16 @@
     "en-US": "as {{cssxref(\"length\")}}",
     "fr": "comme {{cssxref(\"length\")}}",
     "ja": "{{cssxref(\"length\")}} 通り",
-    "ru": "как {{cssxref(\"length\")}}"
+    "ru": "как {{cssxref(\"length\")}}",
+    "zh-C": "作为 {{cssxref(\"length\")}}"
```
Comment (T34-active) [inline l10n/css.json]:
```suggestion
    "zh-CN": "作为 {{cssxref(\"length\")}}"
```

File: l10n/css.json [hunk 15]
```diff
@@ -365,14 +381,16 @@
     "en-US": "as {{cssxref(\"length\")}}",
     "fr": "comme {{cssxref(\"length\")}}",
     "ja": "{{cssxref(\"length\")}} 通り",
-    "ru": "как {{cssxref(\"length\")}}"
+    "ru": "как {{cssxref(\"length\")}}",
+    "zh-C": "为 {{cssxref(\"length\")}} 值"
```
Comment (T34-active) [inline l10n/css.json:385]:
```suggestion
    "zh-CN": "为 {{cssxref(\"length\")}} 值"
```

**Comment by T34-active** [comment]
It's so hard that I wrote so many parts that basically wrong. just one question, How to find the Formal definition CSS property in JSON? @yin1999 @bsmth

**Review by yin1999 — APPROVED** [review]
LGTM for the `zh-CN` translation

**Comment by yin1999** [comment]
> It's so hard that I wrote so many parts that basically wrong. just one question, How to find the Formal definition CSS property in JSON? @yin1999 @bsmth

Search the references of those keys within this repo, or search the English text using Google, or bing, like this: `https://www.google.com/search?q=%22an+absolute+length%3B+if+the+keyword+none+is+specified%2C+the+computed+value+is+0%22+site%3Adeveloper.mozilla.org&oq=%22an+absolute+length%3B+if+the+keyword+none+is+specified%2C+the+computed+value+is+0%22+site%3Adeveloper.mozilla.org`

**Comment by bsmth** [comment]
Thanks a lot! Merging shortly 🚢

**Pull request merged**